File: JeffSebastian/playground/src/main/java/nl/uva/se/ast/expression/logical/Equal.java
package nl.uva.se.ast.expression.logical;

import nl.uva.se.ast.expression.Binary;
import nl.uva.se.ast.expression.Expression;
import nl.uva.se.visitor.Visitor;

public class Equal extends Binary{

	public Equal(int lineNumber, int offset, Expression left, Expression right) {
		super(lineNumber, offset, left, right);
		}

	@Override
	public void accept(Visitor visitor) {
		visitor.visit(this);
	}

}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binary/Equal.cs
using AST.Nodes.Interfaces;

namespace AST.Nodes.Expressions.Binary
{
    public class Equal : BaseBinary
    {
        public Equal(BaseExpression left, BaseExpression right, PositionInText position)
            :base(left, right, position)
        {}

        public override T Accept<T>(ASTVisitors.Interfaces.IExpressionVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }

        public override string ToString()
        {
            return "==";
        }
    }
}
